descartes_light/edge_evaluators/include/descartes_light/edge_evaluators/euclidean_distance_edge_evaluator.h

```diff
@@ -32,7 +32,7 @@ class EuclideanDistanceEdgeEvaluator : public EdgeEvaluator<FloatType>
 {
 public:
   EuclideanDistanceEdgeEvaluator() = default;
-  EuclideanDistanceEdgeEvaluator(const Eigen::Array<FloatType, Eigen::Dynamic, 1>& scale);
+  EuclideanDistanceEdgeEvaluator(Eigen::Array<FloatType, Eigen::Dynamic, 1> scale);
```

Comment:
Same comment about passing `Eigen` things by value:

https://eigen.tuxfamily.org/dox/group__TopicPassingByValue.html

Reply:
Updated to use Eigen::Ref.